examples/minimal/package.json

```diff
@@ -8,12 +8,10 @@
         "build": "tsc"
     },
     "dependencies": {
-        "@pulumi/pulumi": "2.0.0-beta.3"
+        "@pulumi/pulumi": "^2.0.0",
+        "@pulumi/vsphere": "^2.0.0"
     },
     "devDependencies": {
         "typescript": "^2.6.2"
-    },
-    "peerDependencies": {
-        "@pulumi/vsphere": "dev"
```

Comment:
Same question about the `peerDependencies`

Reply:
This isn't correct - this means we don't actually get an upgrade test - we ad to change this in GCP to make this work correct to move away from peerDependencies - this will be being done for all of the providers